services/app-api/utils/formTemplates/formTemplates.test.ts

```diff
@@ -23,19 +29,30 @@ const currentMCPARFormHash = createHash("md5")
   .update(JSON.stringify(mcpar))
   .digest("hex");
 
+const pccmTemplate = generatePCCMTemplate(mcpar);
+const currentPCCMFormHash = createHash("md5")
+  .update(JSON.stringify(pccmTemplate))
+  .digest("hex");
+
 describe("Test getOrCreateFormTemplate MCPAR", () => {
   beforeEach(() => {
     jest.restoreAllMocks();
   });
   it("should create a new form template if none exist", async () => {
+    // mocked once for search by hash
+    mockDocumentClient.query.promise.mockReturnValueOnce({
+      Items: [],
+    });
```

Comment:
test: added mocks to many tests because we now have two queries: one for matching hash, and one for latest version